app/webpacker/css/admin_v3/globals/palette.scss

```diff
@@ -1,7 +1,10 @@
 // Basic color palette for admin styles v3
-$color-1:                        #FFFFFF !default;    // White
-$color-2:                        #9FC820 !default;    // Green
-$color-3:                        #008397 !default;    // Teal (Allports)
-$color-4:                        #6788A2 !default;    // Dark Blue
-$color-5:                        #C85136 !default;    // Red/Orange (Mojo)
-$color-6:                        #FF9300 !default;    // Yellow
+$color-1: #ffffff !default; // White
+$color-2: #9fc820 !default; // Green
+$color-3: #008397 !default; // Teal (Allports)
+$color-4: #6788a2 !default; // Dark Blue
+$color-5: #c85136 !default; // Red/Orange (Mojo)
+$color-6: #ff9300 !default; // Yellow
+$color-7: #eff1f2 !default; // Light grey
+$color-8: #191c1d !default; // Near-black
+$color-9: #2e3132 !default; // Dark Grey
```

Comment:
Maybe it's difficult in the transition between styles but I find it hard to read `$color-7` in CSS without knowing which one it is.